components/clarity-vscode/README.md

```diff
@@ -1,50 +1,69 @@
-# Clarity for VSCode
+# Clarity for Visual Studio Code
 
-Clarity is a decidable smart contract language that optimizes for predictability and security. Smart contracts allow developers to encode essential business logic on a blockchain.
+Clarity is a **decidable** smart contract language that optimizes for predictability and security. Developed by Hiro, Clarity enables smart contract developers to encode essential business logic on a blockchain.
 
-This extension aims to help developer write safe and clean Clarity code, it includes syntax highlighting, auto-completion, linting, debugging and [safety checks](https://www.hiro.so/blog/new-safety-checks-in-clarinet),.
+A programming language is referred to as **decidable** if you can know with certainty, from the code itself, what the program will do. Clarity is was designed to be intentionally Turing incomplete, thereby avoiding the problem of `Turing complexity` (the number of steps it takes for a turing machine to run an algorithm on a given input size). This allows for complete static analysis of the entire call graph of a given smart contract, while Hiro's support for types and type checkers can eliminate whole classes of bugs like unintended casts, reentrancy bugs, and reads of uninitialized values.
 
-## Support for [vscode.dev](https://vscode.dev)
+The Language Server Protocol (LSP) defines the protocol used between an editor or IDE, and a language server that provides language features such as auto-complete, go-to-definition, find-all-references, and others.
```

Comment:
We probably don't need to get into details about the LSP is

Reply:
Agreed.

Reply:
removed from intro section